synapse/state/__init__.py

```diff
@@ -129,6 +130,12 @@ def __init__(self, hs: "HomeServer"):
         self.hs = hs
         self._state_resolution_handler = hs.get_state_resolution_handler()
         self._storage_controllers = hs.get_storage_controllers()
+        self._events_shard_config = hs.config.worker.events_shard_config
+        self._instance_name = hs.get_instance_name()
+
+        self._update_current_state = (
```

Comment:
Might be worth naming this `_client` to make it obvious what the difference is between it and `update_current_state`?

Reply:
Renamed!